test/LibraryService.test.ts

```diff
@@ -0,0 +1,46 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../src/config/serverConfig';
+import { cleanUpDb, testSetup } from './utils';
+const supertest = require('supertest');
+
+let server: Server;
+
+const Library: fhir4.Library = {
+  resourceType: 'Library',
+  type: { coding: [{ code: 'logic-library' }] },
+  id: 'library123',
+  status: 'active'
+};
+
+describe('LibraryService', () => {
+  beforeAll(async () => {
+    const config = serverConfig;
+    server = initialize(config);
+    await testSetup([Library]);
+  });
+  describe('CRUD operations', () => {
```

Comment:
Idk if it makes sense to group this test as `CRUD operations`. Kinda misleading since this server is probably only going to support the `R` part of `CRUD`. Maybe we rename this describe block to just be the function that it's testing: `searchById`?

```suggestion
  describe('searchById', () => {
```